### PR TITLE
Article: Foundation

### PR DESCRIPTION
**Pull request by theflametrooper**
Refer #707

### CODE REVIEW

File: Foundation.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Foundation
+
+Foundation is the most advanced responsive front-end framework in the world (According to their website).
```
Comment (alayek) [inline Foundation.md]: Lol! But seriously, change it something like

> Foundation is one of the most advanced responsive front-end frameworks.

And also link the word _Foundation_ to the homepage of Foundation.

**Comment by alayek** [comment]
This is similar to Bootstrap.md file, so you know how to modify this later.

# LGTM :+1:

**Comment by abhisekp** [comment]
@theflametrooper **File Rename:-**
```diff
- Foundation.md
+ CSS-Framework-Foundation.md
```

File: Foundation.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  </head>
+  <body>
+    <h1>Hello World</h1>
+    <!-- Add all HTML Code here -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/foundation/6.2.1/foundation.min.js"></script>
+  </body>
+</html>
+```
+
```
Comment (abhisekp) [inline Foundation.md]: Add a http://jsbin.com link

<kbd><a href="http://jsbin.com"> <strong>JSBin Demo ⮭</strong></a></kbd>

```md
<kbd><a href="http://jsbin.com"><img src="http://i.imgur.com/wBLOt8L.png"> <strong>JSBin Demo ⮭</strong></a></kbd>
```

File: Foundation.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+We have used a CDN in this example, but you can checkout other ways of installing Foundation [here](http://foundation.zurb.com/sites/download/).
+
+## Learning Resources
+
+- The official documentation for Foundation is available [here](http://foundation.zurb.com/sites/docs/).
+- Checkout Foundation's open source GitHub repository [here](https://github.com/zurb/foundation-sites).
+- [Here](http://foundation.zurb.com/learn/tutorials.html) is a list of tutorials on using the Foundation Framework.
```
Comment (abhisekp) [inline Foundation.md]: - :point_right: List of [**`Foundation Framework tutorials`**](http://foundation.zurb.com/learn/tutorials.html)

**Pull request merged**